mix.exs

```diff
@@ -1,11 +1,26 @@
 defmodule Trento.MixProject do
   use Mix.Project
 
+  @version "1.1.0"
+  defp get_version do
+    case System.get_env("VERSION") do
```

Comment:
you dont need a case here, just `System.get_env("VERSION", get_version_from_git())` should work

Reply:
Thanks! I actually did it this way at some point... I guess too many back and forths hehe